jdbc/src/test/java/nextstep/jdbc/DataAccessUtilsTest.java

```diff
@@ -0,0 +1,44 @@
+package nextstep.jdbc;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class DataAccessUtilsTest {
+
+    @DisplayName("결과값이 비어있을 경우 예외가 발생한다.")
```

Comment:
```suggestion
    @DisplayName("결과값이 존재하는 경우 해당 값이 반환된다.")
```
이런건 어떨까요? 예외가 발생하는 상황이 아니네요 😄